Extended README with contribution information

javagl opened the pull request:
This addresses https://github.com/KhronosGroup/glTF-Project-Explorer/issues/14

Preview at https://github.com/javagl/glTF-Project-Explorer#contributions 

Comments or suggestions?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KhronosGroup/glTF-Project-Explorer?pullRequest=15) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KhronosGroup/glTF-Project-Explorer?pullRequest=15) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KhronosGroup/glTF-Project-Explorer?pullRequest=15) it.</sub>

weegeekps commented on README.md:
```diff
@@ -4,6 +4,50 @@ Tool to provide a filterable registry of glTF community projects.
+|------|--------|-------------|
+| `name` | `string` | The name of the project. This will be the title of the project card, and turned into a link, leading to the URL that is given as the `link` property |
+| `link` | `string` (URL) | The URL of the homepage of the project.  |
+ | `description` | `string` | A short description of the project. This string can contain basic [Markdown](https://en.wikipedia.org/wiki/Markdown) formatting. |
```
The parser we're using supports GitHub flavored Markdown, so the [documentation Github provides](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) may be more useful than the Wikipedia page.

weegeekps commented on README.md:
````diff
@@ -4,6 +4,50 @@ Tool to provide a filterable registry of glTF community projects.
+All properties except for the `name` are optional. 
+
+An example of the JSON description of a project is shown here:
+ ```
````
It might be good to add `json` to the end of this line to enable the syntax highlighting for readability.

javagl commented:
I have added syntax highlighting - it shows properly in Atom, but not on Github... (?)

I also updated markdown link. By the way, I noticed that there are some errors in the data file related to that: In the `input`/`output` fields, some projects did have a markdown link saying "Multiple", linking to a site where the formats have been listed. This link was broken during the conversion (and I'll go though that ASAP). 

But it raises one more general question: Which fields should allow markdown? I think restricting it to **only** allow it in the description makes sense. (If someone wants to add a link the the supported `input`/`output` formats, for example, it can be added there).

weegeekps commented:
Right now we only parse the markdown within the description, and I agree that restricting it to there makes sense. I think it would also make sense to make a validator script that we can run on PRs using GitHub Actions to require all PRs containing data file modifications to be valid. I'm going to add an issue for this; I don't think we'll get to this soon but it seems useful.

weegeekps commented:
Also, I don't know how we want to handle merges? I figure approver handles the merge? Let me know.

javagl commented:
Assuming that the latter refers to the "external" PRs with updates of the data file: I think that generally, there shouldn't be much to "review", so approve+merge could be done quickly. 

The point about some sort of validity check also came to my mind. On the one hand, I'm not sure whether setting up the infrastructure for that is worth the effort: There may be a few updates when the explorer gains some visibility, and many people update their project information. But after that, I think there may not be much more than 1-2 new projects per week...? On the other hand: Considering that the JSON data file is "live", a broken file (due to some JSON parsing error caused by a missed `,` or `"`) would be critical and might immediately make the page inaccessible...
